### DISCUSSION TITLE
can't add a new keyboard layout language.

### DISCUSSION THREAD

**Discussion by YousefSaber** [lxqt/lxqt#2142] [Q&A] [answered]
on Fedora Workstation 35 I can't add a new Keyboard layout language does anyone know how to do it manually?

**Comment by stefonarch** [comment, marked as answer]
Can you describe more explicitly? In "mouse and keyboard"  > keyboard layout > add ?

In qterminal you can use the comand `setxkbmap us,ru,de` (example) but this is not permanent.

**Reply by YousefSaber** [reply]
@stefonarch 

this command fixed the issue for me thank you very much, I can now add and remove keyboard layouts to my system after running this command I've added the Arabic language the desired language I wanted to add 

```
[jsaber@fedora ~]$ setxkbmap -v 10 us,ru,de
Setting verbose level to 10
locale is C
Warning! Multiple definitions of keyboard layout
         Using command line, ignoring X server
Trying to load rules file ./rules/evdev...
Trying to load rules file /usr/share/X11/xkb/rules/evdev...
Success.
Applied rules from evdev:
rules:      evdev
model:      pc105
layout:     us,ru,de
Trying to build keymap using the following components:
keycodes:   evdev+aliases(qwerty)
types:      complete
compat:     complete
symbols:    pc+us+ru:2+de:3+inet(evdev)
geometry:   pc(pc105)
```

by the way looking at the output of this command what was the error in my system? 
>    notice that before running this command I had first to install `setxkbmap`

**Reply by stefonarch** [reply]
I don't know Fedora's setup, if I try to remove x11-xkb-utils it would remove xserver. 
Probably the issue was that setxkbmap missing.
